environments/kubernetes/main.jsonnet

```diff
@@ -1,11 +1,13 @@
 local app =
   (import 'kube-thanos.libsonnet') +
-  (import 'telemeter.libsonnet');
+  (import 'telemeter.libsonnet') +
+  (import '../../components/jaeger-collector.libsonnet');
```

Comment:
Can we create `jaeger-collector.libsonnet` where we import `../../components/jaeger-collector.libsonnet` so that we can do the overrides for Kubernetes there.
In the future the `../../components/jaeger-collector.libsonnet` should come from somewhere upstream.

Reply:
Thanks for the clarification! I've noticed the pattern, but wasn't sure what's the purpose.

Reply:
How does it look now?